app/logic/Calendar/Event.ts

```diff
@@ -1,4 +1,3 @@
-import { PersonUID, findPerson } from "../Abstract/PersonUID";
```

Comment:
I can't do this as a suggestion, but unsurprisingly `Event` uses the `Participant` type.
```typescript
import type { Participant } from "./Participant";
```

Reply:
Fixed in git commit ddde84ce